app/controllers/api/v1/users_controller.rb

```diff
@@ -169,7 +169,9 @@ def create_user_params
       end
 
       def update_user_params
-        @update_user_params ||= if external_auth?
+        @update_user_params ||= if external_auth? && current_user.role.name.eql?('Administrator')
+                                  params.require(:user).permit(:name)
+                                elsif external_auth?
```

Comment:
Greenlight is a bit more tricky than this. The "Administrator" role can be renamed, or the permission to edit users can be given to other roles, meaning that just this check isn't enough. What you need to do instead is check if the user has the `ManageUsers` permission

Reply:
Here's how you would do that:

`PermissionsChecker.new(current_user:, permission_names: 'ManageUsers', current_provider:).call`